src/app/view/timestamp-editor/timestamp-editor.component.html

```diff
@@ -23,14 +23,14 @@ <h4>For Transport-Call {{ transportCall.transportCallSequenceNumber}}
       </div>
       <div class="md:col-4" *ngIf="respondingToTimestampInfo != null">
         <!-- Placeholder for additional information when rejecting timestamp -->
-        <div *ngIf="parseTimestampResponseStatus() == 'ACCEPTED'">
+        <div *ngIf="timestampResponseStatus == TimestampResponseStatus.ACCEPT">
```

Comment:
Here we only use `==` but in the other `ngIf`s we use `===`.  I think we should be consistent in using either 2 or 3 (to avoid people wondering if there is a reason for the difference).